<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt RSA Demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QListWidget" name="listWidget"/>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QTextEdit" name="textEditPrivateKey">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QTextEdit" name="textEditPublicKey">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QTextEdit" name="textEdit"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>585</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGenerate"/>
   <addaction name="actionEncode"/>
   <addaction name="actionDecode"/>
  </widget>
  <action name="actionGenerate">
   <property name="text">
    <string>Generate</string>
   </property>
   <property name="toolTip">
    <string>Generate RSA Key</string>
   </property>
  </action>
  <action name="actionEncode">
   <property name="text">
    <string>Encode</string>
   </property>
  </action>
  <action name="actionDecode">
   <property name="text">
    <string>Decode</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
